packaging/os/zypper_repository.py

```diff
@@ -129,9 +129,26 @@
 
 from distutils.version import LooseVersion
 
+def _get_cmd(subcommand, *args):
+    """Combines the non-interactive zypper command with subcommands and arguments"""
+    cmd = ['/usr/bin/zypper', '--quiet', '--non-interactive']
+
+    is_listrepo = subcommand == "repos"
```

Comment:
The `is_listrepo` is not necessary, just pass all the args directly:

```
_get_cmd(*args):
    cmd = ['/usr/bin/zypper', '--quiet', '--non-interactive']
    cmd.extend(args)
    return cmd
```

And then `_get_cmd('--xmlout', 'repos')`